pygbx/gbx.py

```diff
@@ -80,6 +80,11 @@ def __init__(self, obj):
             self.root_parser = ByteReader(obj)
 
         self.magic = self.root_parser.read(3, '3s')
+        try:
+            self.magic.decode('utf-8')
```

Comment:
Because magic is used only temporarily in the constructor, drop `self` and reuse it:
```
magic = self.root_parser.read(3, '3s')
try:
    magic = magic.decode('utf-8')
except:
    raise GbxLoadError(f'obj is not a valid Gbx data: can not decode unicode')

if magic != 'GBX':
   ...

Reply:
Good one. Pushed another version, which includes this change.